content/en/dynamic_instrumentation/_index.md

```diff
@@ -23,118 +23,162 @@ further_reading:
 
-### Creating a probe
+# Using Probes
+
+A probe allows you to collect data from specific points in your code without halting the execution of the program. 
+
+If you're a SRE, you can think of probes as a way of enhancing your observability by adding dynamic logs, metrics, and spans to running application without needing to stop or restart anything. This meant you can gather data immediately without disturbing the user experience or requiring lengthy deployments.
 
-Both log and metric probes require the same initial setup:
+If you're a developer, you can think of a probe as a "non-breaking breakpoint". In traditional debugging, a breakpoint is a point in the program where the execution stops, allowing the developer to inspect the state of the program at that point. However, in real-world production environments, it's not practical or even possible to stop the execution of the program. Probes fill in this gap by allowing you to inspect variable state in production environments in a completely non-intrusive way.
+
+## Creating a probe
+
+All probe types require the same initial setup:
 
 1. Go to the [Dynamic Instrumentation page][12].
 1. Click **Create Probe** in the top right, or click the three dot context menu on a service and select **Add a probe for this service**.
-1. If not prefilled, choose a service from the list.
-1. If not prefilled, choose runtime, environment and version.
+1. If not prefilled, choose service, runtime, environment and version.
 1. In the source code, specify where to set the probe by selecting either a class and method or a source file and line. 
    If you set up Source Code Integration for your service, autocomplete shows suggestions for the selecting a file and displays the file's code so you can choose the line.
 
-### Creating a log probe
+## Probe Types
+
+### Log Probes
+
+A *log probe* emits a log when it executes. 
 
-A *log probe* emits a log when it executes.
+Log probes are enabled by default on all service instances that match the specified environment and version. They are rate limited to execute at most 5000 times per second, on each instance of your service.
 
-If you enable `Capture method parameters and local variables` on the log probe, it also captures the following values from the execution context and adds them to the log event:
-- method arguments
-- local variables
-- class fields
-- the call stack
-- exceptions 
-You can see the captured values in the Datadog UI. 
+If you enable `Capture method parameters and local variables` on the log probe, the following debugging data will be captured and added it to the log event:
+  - **Method arguments**, **local variables**, and **fields**, with the following limits by default:
+    - Follow references three levels deep (configurable in the UI).
```

Comment:
configurable up to five levels?

Reply:
I think this UX is still up for debate, so prefer leaving that unspecified for now